addons/crm/views/crm_lead_views.xml

```diff
@@ -566,6 +566,8 @@
                     <field name="probability"/>
                     <field name="active"/>
                     <field name="company_currency"/>
+                    <field name="expected_revenue"/>
+                    <field name="recurring_revenue"/>
```

Comment:
Does this issue happen in master actually? Feels strange to have gone unnoticed? I could not reproduce it anymore ^^ so this is not linked to your changes, right?